flow/layers/transform_layer.cc

```diff
@@ -11,6 +11,17 @@ TransformLayer::TransformLayer() = default;
 TransformLayer::~TransformLayer() = default;
 
 void TransformLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
+  // Checks (in some degree) that SkMatrix transform_ is valid and initialized.
```

Comment:
Can the `TransformLayer` constructor initialize the `transform_` to the identity matrix?

Another option would be setting a flag when `set_transform` is called and checking that flag here.

Reply:
I'll add a flag, together with this isFinite check. The reason that we still need this isFinite check is that someone could call set_transform with an uninitialized SkMatrix.